setupext.py

```diff
@@ -1817,8 +1811,6 @@ def add_flags(self, ext):
             (tcl_lib_dir, tcl_inc_dir, tcl_lib,
              tk_lib_dir, tk_inc_dir, tk_lib) = result
             ext.include_dirs.extend([tcl_inc_dir, tk_inc_dir])
-            ext.library_dirs.extend([tcl_lib_dir, tk_lib_dir])
-            ext.libraries.extend([tcl_lib, tk_lib])
```

Comment:
I adding a minimal set of defines from TCL / Tk in the latest commit.  It seems to compile and run correctly on OSX, Linux and Windows, but I would appreciate some eyes on the minimal header, because of my aforementioned C-fu weakness.

If this new header is OK, then I'll go ahead and delete all the TCL / Tk finding logic in `setupext.py`.